test/examples/30_pybind11_examples/02_advanced/05_type_conversions/03_functional/chimera_boost_python.yaml

```diff
@@ -0,0 +1,7 @@
+# Arguments that will be appended in-order before command line arguments.
+arguments:
+  - "-extra-arg"
+  - "-I/usr/lib/clang/3.6/include"
+namespaces:
+  'chimera_test':
+    name: null # TODO: otherwise, import error
```

Comment:
Is there already an issue filed against this?  We probably just need to slightly modify the place where this lookup happens to correctly assign null if the "name" is not set.

Reply:
Created one: https://github.com/personalrobotics/chimera/issues/299